tests/integrationTests/inputs/volumeOfFluids/twoGasAdvectingDiscontinuity.yaml

```diff
@@ -17,8 +17,8 @@ timestepper:
     upper: [ 10 ]
     boundary: [ NONE ]
     modifiers:
-      - !<!ablate::domain::modifiers::GhostBoundaryCells> ""
       - !<!ablate::domain::modifiers::DistributeWithGhostCells> ""
+      - !<!ablate::domain::modifiers::GhostBoundaryCells> ""
```

Comment:
Would you mind updating this yaml file to get rid of the extra !< around each tag and using names instead of numbers for the anchors?